netjsonconfig/backends/openwrt/converters/wireless.py

```diff
@@ -333,7 +333,7 @@ def __fix_netjson_network(self, result):
                     del result[index]['network']
         return result
 
-    def _track_bridged_wifi(self, intermediate_data=None):
+    def _track_bridged_wifi(self, intermediate_data=None):  # noqa: C901
```

Comment:
this should not be done

Reply:
Thanks :+1: , I checked your commit. I will not use `noqa: C901` in future.